client/my-sites/plans/main.jsx

```diff
@@ -379,6 +384,7 @@ class Plans extends Component {
 
 		const currentPlanSlug = selectedSite?.plan?.product_slug;
 		const isEcommerceTrial = currentPlanSlug === PLAN_ECOMMERCE_TRIAL_MONTHLY;
+		const isMigrationTrial = currentPlanSlug === PLAN_MIGRATION_TRIAL_MONTHLY;
```

Comment:
Maybe we want make this variable more generic for business trial, maybe we want to change it to `isBusinessTrial` here?